garden_ai/container/containerize.py

```diff
@@ -113,14 +119,24 @@ def start_container(
         run_command.insert(idx, entrypoint)
         run_command.insert(idx, "--entrypoint")
 
-    subprocess.run(run_command)
+    output = subprocess.run(run_command, capture_output=True, text=True)
+
     if cleanup:
         subprocess.run(["docker", "rmi", image_name])
 
+    return output.stdout
+
+
+def garden_pipeline(model_connectors=None):
```

Comment:
ooh this is one of my favorite python footguns: 
```
# both of these work the same, `myfunc` gets bound to the decorated function 
@garden_pipeline(model_connectors=None)
@garden_pipeline() 
def myfunc(): 
    ...
# breaks silently, bc `myfunc` gets bound to `garden_pipeline.<locals>.decorator`
@garden_pipeline 
def myfunc():
    ... 
```
check out the pattern I used for the `step` decorator, that's the least-kludgy way I know of getting `@foo()` and `@foo` to behave consistently